cmd/mr_approve.go

```diff
@@ -48,25 +54,17 @@ var mrApproveCmd = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		if comment || len(msgs) > 0 || filename != "" {
-			linebreak, err := cmd.Flags().GetBool("force-linebreak")
+		note := comment || len(msgs) > 0 || filename != ""
+		linebreak := false
+		if note {
+			linebreak, err = cmd.Flags().GetBool("force-linebreak")
 			if err != nil {
 				log.Fatal(err)
 			}
-
-			createNote(rn, true, int(id), msgs, filename, linebreak, "")
 		}
 
-		err = lab.MRApprove(p.ID, int(id))
-		if err != nil {
-			if err == lab.ErrStatusForbidden {
-				log.Fatal(err)
-			}
-			if err == lab.ErrActionRepeated {
-				fmt.Printf("Merge Request !%d already approved\n", id)
-				os.Exit(1)
-			}
-		}
+		msgs = append(msgs, "/approve")
```

Comment:
The `msg` var is not used when the user calls for a filename `-F`; shouldn't the quick action be added to the end of the file content when the user saves and closes it so he still get the "approve" action with the file content is used as the omment? (the same is valid for the unapprove code)